README.md

```diff
@@ -34,7 +34,7 @@ In your original terminal window, at the top level of your folder, initialize th
 and follow the prompts:
 
 ```javascript
-openzeppelin init
+npx openzeppelin init
```

Comment:
I'd rename these to `oz`, both because that's the usage we feature on the docsite, and so that this isn't mixed up with the `@openzeppelin/cli` command above.

Reply:
I have updated to use the `oz` alias